packages/config/src/env/envelope.js

```diff
@@ -0,0 +1,25 @@
+import { throwUserError } from '../error.js'
+import { ERROR_CALL_TO_ACTION } from '../log/messages.js'
+
+export const getEnvelope = async function ({ api, accountId, siteId }) {
+  if (accountId === undefined) {
+    return {}
+  }
+  try {
+    const environmentVariables = await api.getEnvVars({ accountId, siteId })
+    // eslint-disable-next-line fp/no-mutating-methods
+    const envVars = environmentVariables
+      .sort((left, right) => (left.key.toLowerCase() < right.key.toLowerCase() ? -1 : 1))
+      .reduce((acc, cur) => {
+        const envVar = cur.values.find((val) => ['dev', 'all'].includes(val.context))
+        if (envVar && envVar.value) {
+          // eslint-disable-next-line fp/no-mutation, no-param-reassign
+          acc[cur.key] = envVar.value
```

Comment:
I think this should make the linter happy?

```suggestion
          return {
            ...acc,
            [key]: value
          }
```

Reply:
Good catch!